### PR TITLE
Add soviet-02a

### PR DESCRIPTION
**Pull request by Computerfreaked**
Soviet 02a mission as it also is on resource site

### CODE REVIEW

**Comment by Mailaender** [comment]
You also need to add it to https://github.com/OpenRA/OpenRA/blob/bleed/mods/ra/missions.yaml#L7

File: mods/ra/maps/soviet-02a/soviet02a.lua [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+		Trigger.OnEnteredFootprint(waypoint7location, function(unit, id)
+			Trigger.RemoveFootprintTrigger(id)
+			
+			--for each unit in CmdAtk
+			Utils.Do(CmdAtk, function(unit)
+				if not unit.IsDead then
+					--remove the attackmove trigger
+					--Trigger.Clear(unit, "OnIdle")
+
+					--TODO: Order the unit to attack the commandcenter, there is currently no command to do so
```
Comment (Mailaender) [inline mods/ra/maps/soviet-02a/soviet02a.lua]: `unit.Attack(Actor43)`

See https://github.com/OpenRA/OpenRA/wiki/Lua-API-(playtest)#actor-properties--commands

File: mods/ra/maps/soviet-02a/soviet02a.lua [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+	Trigger.OnPlayerLost(player, function()
+		Media.PlaySpeechNotification(player, "MissionFailed")
+	end)
+
+	--camera starting position (in original game always at waypoint98)
+	Camera.Position	= waypoint98.CenterPosition
+	
+	--if the command center is destroyed player loses
+	--then it is destroyed player looses
+	Trigger.OnAnyKilled({ Actor43 }, function (dontcare)
```
Comment (Mailaender) [inline mods/ra/maps/soviet-02a/soviet02a.lua]: We also have `Trigger.OnKilled(Actor43, ...`

File: mods/ra/maps/soviet-02a/soviet02a.lua (hunk 1 above)
Comment (abcdefg30) [inline mods/ra/maps/soviet-02a/soviet02a.lua]: "Fly an attack against the target actor. **Requires Trait:** AttackPlane "

**Comment by abcdefg30** [comment]
@Computerfreaked stated that he won't be able to work on this. So I started working on it [here](https://github.com/OpenRA/OpenRA/compare/bleed...abcdefg30:soviet02a).

File: mods/ra/maps/soviet-02a/soviet02a.lua (hunk 1 above)
Comment (Computerfreaked) [inline mods/ra/maps/soviet-02a/soviet02a.lua]: https://github.com/OpenRA/OpenRA/issues/7453

Comment (Mailaender) [inline mods/ra/maps/soviet-02a/soviet02a.lua]: Ugh sorry. Use `unit.AttackMove(Actor43.Location)` then.

**Comment by Computerfreaked** [comment]
That is correct abcdefg30. I have too less knowledge of Github and to little time unfortunately :(

**Comment by abcdefg30** [comment]
I filed #8285.

**Pull request closed without merging**